larpandora/LArPandoraInterface/LArPandoraGeometry.cxx

```diff
@@ -262,6 +277,18 @@ void LArPandoraGeometry::LoadGeometry(LArDriftVolumeList &driftVolumeList)
             UIntSet tpcList;
             tpcList.insert(itpc1);
 
+            if (isDualPhase)
+            {
+                LArDaughterDriftVolumeList tpcVolumeList;
+                tpcVolumeList.push_back(LArDaughterDriftVolume(icstat, itpc1));
+
+                driftVolumeList.push_back(LArDriftVolume(driftVolumeList.size(), isPositiveDrift,
```

Comment:
Recommend using `std::vector<T>::emplace_back(...)` instead of `std::vector<T>::push_back(T(...))`, and elsewhere below.